stats/stats_test.go

```diff
@@ -168,6 +169,34 @@ func TestHistogram(t *testing.T) {
 	}
 }
 
+func XTestHistogramData(t *testing.T) {
```

Comment:
cc @olimpias - this is what I meant/expect
I will add the code now that I have a test :-)

Reply:
Now I see, Thanks for example :).